Geometry/TrackerNumberingBuilder/interface/utils.h

```diff
@@ -0,0 +1,14 @@
+#ifndef GEOMETRY_TRACKERNUMBERINGBUILDER_UTILS_H
+#define GEOMETRY_TRACKERNUMBERINGBUILDER_UTILS_H
+
+#include "Geometry/TrackerNumberingBuilder/interface/GeometricDet.h"
+
+/**
+ * A helper method to get the full list of SiStrip DetIds from the GeometricDet
+ *
+ * The DetIds are sorted by subdetector, but otherwise keep the ordering from
+ * GeometricDet::deepComponents (for compatibility with SiStripDetInfoFileReader)
+ */
+std::vector<uint32_t> getSiStripDetIds(const GeometricDet& geomDet);
```

Comment:
Please put this utility in a namespace. See [7.13](https://github.com/cms-sw/cms-sw.github.io/blob/master/cms_coding_rules.md#7----design-and-coding-guidelines-1) of our coding guidelines.